d-rita/fast-food-fast-react
# #164353021 Users should be able to sign up

#### What does this PR do?
This PR enables users to sign up with fast-food-fast.

#### Description of tasks to be completed 
- Add auth reducer
- Add register actions
- Add register form

#### How to manually test this?
- Fetch and check out this branch using `git fetch origin ft-signup-164353021 && git checkout ft-signup-164353021`
- To install dependencies, use `yarn install`
- Start server with `yarn start`
- Go to `https://localhost:8080/` and register

#### Related Pivotal Tracker Story
[#164353021](https://www.pivotaltracker.com/story/show/164353021)
